### ISSUE TITLE
✨ Admin / Skill families / mostly Styling update

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#11477]
## ✨ Feature

Update the skill families page to the new admin design patterns

### Out of scope for this issue (don't build this even if it's in the design)
- [ ] Moving to the new nav
- [ ] Updates to the table component
- [ ] The "advanced tools" tab and everything in it
- [ ] Search/filter dialog up top (don't worry about it)

## 🎨 Design

[Figma files](https://www.figma.com/design/W2cVcRK5YtrllE0sbVgwEV/Admin---Skills-families?node-id=14051-2833&t=gOuxokYXQXHWhokM-1) by @Jerryescandon 

## ✅ Acceptance Criteria

- [ ] Skill families page (/admin/settings/skill-families)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Remove the "edit" link column from the table and add links on the "name" column to bring users to the "information" page
- [ ] Create skill family (/admin/settings/skill-families/create)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling

- [ ] New page - Skill families information (/admin/skill-families/:id:)
  - [ ] Header  (background, H1 & breadcrumb)
  - [ ] H2 and content styling
  - [ ] Only one tab for now
  - [ ] Inline form for displaying information and editing
- [ ] Edit skill family page (/admin/skill-families/:id:/edit)
  - [ ] (should use same component as Create Skill families page)

- [ ] mobile view for all pages

- [ ] Design Review by @Jerryescandon 

## Blockers
```[tasklist]
### Blocked by
- [ ] #11344 
```

**Comment by gobyrne** [comment]
@NienkeBr well at this point I'm just tagging you on everything in the milestone
